### PR TITLE
Tagging - Cache required items check and use uniqueItems

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- title

### CODE REVIEW

File: addons/tagging/functions/fnc_checkTaggable.sqf [hunk 1]
```diff
@@ -36,7 +38,6 @@
     // Exit if trying to tag a non static object
     TRACE_1("Obj:",_intersections);
 
-    // Exit if trying to tag a non static object
```
Comment (dedmen) [inline addons/tagging/functions/fnc_checkTaggable.sqf]: Why remove?

Comment (mharis001) [reply addons/tagging/functions/fnc_checkTaggable.sqf]: Duplicate comment as above

Comment (dedmen) [reply addons/tagging/functions/fnc_checkTaggable.sqf]: 🤦‍♂️ oof

**Comment by PabstMirror** [comment]
Added a commit that ensures `_requiredItem` is always config-case, should let us drop all the `apply {toLower _x}`

**Pull request merged**